core/src/main/java/me/lihq/game/ScenarioBuilder.java

```diff
@@ -119,6 +120,9 @@ Set the first clues in the list to red herrings (the number of red herrings
 
             if (randHidingSpot != null) {
                 room.addClue(clue.setTileCoordinates(randHidingSpot));
+            } else
+            {
+                System.out.println("Failed to place Clue " + clue.getName() + " in room " + room.getName());
```

Comment:
It is quite crucial, cause then a clue isnt placed in the map, could be the murder weapon. Therefore the game isn't completable. I'll change it to an exception